HISTORY.rst

```diff
@@ -7,6 +7,8 @@ Release History
 **Features and Improvements**
 
 - Cover Python 3.9 and 3.10 and Django 3.2, drop Python 3.5 and Django 3.0 support
+- Bump Behave requirement to 1.2.7 (allows option to change the Behave TestCase)
+- New option to change the Django TestCase
```

Comment:
Isn't this really an option to change the test runner? We don't supply a custom [test case class](https://github.com/behave/behave-django/blob/main/behave_django/testcase.py) with the new option, do we?

Reply:
Crud. I got my test classes mixed up, yes, these are TestCase, not TestRunner. I'll change the names now.

Reply:
Ok, can you let this last commit run tests, let's see if it passes *fingers crossed*

Reply:
Alright. The pipeline passes. Let's merge the changes?

Reply:
Push it real good ;)